apps/src/templates/Notification.jsx

```diff
@@ -145,9 +146,10 @@ class Notification extends Component {
     dismissible: PropTypes.bool.isRequired,
     onDismiss: PropTypes.func,
     newWindow: PropTypes.bool,
-    // analyticId is only posted when a primary button is provided.
+    // analyticId and analyticsData are only used when a primary button is provided.
     // It's not used by the array of buttons.
     analyticId: PropTypes.string,
+    analyticsData: PropTypes.object,
```

Comment:
Maybe I should make this something like `firehoseData`, and change the existing `analyticId` to `googleAnalyticsId`?

Reply:
If it's not too much trouble, it'd probably be helpful to rename.

Reply:
It looks like `analyticId` is only used a couple of places, so should be pretty easy.